### PR TITLE
Initial import of Galculator snap.

### PR DESCRIPTION
**Pull request by flexiondotorg**
Here is the initial Galculator snap. Thanks to @didrocks for the help.

### CODE REVIEW

**Comment by dholbach** [comment]
Can you maybe include a `README.md` file and update the top-level `README.md` as well? (https://github.com/ubuntu/snappy-playpen/blob/master/CONTRIBUTING.md has some more info)

**Comment by caldav** [comment]
Can you add a README? (cf.https://github.com/ubuntu/snappy-playpen/blob/master/template.README.md )

**Comment by dplanella** [comment]
Also this morning we were talking about https://code.launchpad.net/~didrocks/+junk/galculator-snap, which has got a simpler wrapper with precompiled binaries for e.g. font support. Have you seen that one?

**Comment by dholbach** [comment]
Can you maybe change add galculator to the top-level `README.md` too?

File: README.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ This project currently includes the following snaps:
 | :white_check_mark:  | `atom`             | [atom-cwayne][atom]       |
 | :white_check_mark:  | `dosbox`           |                           |
 | :white_check_mark:  | `ffmpeg`           |                           |
+|                     | `galculator`       |                           |
```
Comment (dholbach) [inline README.md]: I think you can add `:white_check_mark:` for this one.

**Comment by dholbach** [comment]
Nice work!

**Pull request merged**